sources/packages/web/src/types/contracts/aest/roles.ts

```diff
@@ -30,4 +30,5 @@ export enum Role {
   InstitutionCreateNote = "institution-create-note",
   InstitutionApproveDeclineDesignation = "institution-approve-decline-designation",
   InstitutionApproveDeclineOfferingChanges = "institution-approve-decline-offering-changes",
+  StudentReissueMSFAA = "student-reissue-msfaa",
```

Comment:
What about adding `<check-permission-role>`  in the vue side. I know the Vue button is inside form.io. But is there a way to disable the button when the user does not have right permission?